core/test/results/sample_v2.json

```diff
@@ -4851,14 +4851,14 @@
       "id": "efficient-animated-content",
       "title": "Use video formats for animated content",
       "description": "Large GIFs are inefficient for delivering animated content. Consider using MPEG4/WebM videos for animations and PNG/WebP for static images instead of GIF to save network bytes. [Learn more about efficient video formats](https://developer.chrome.com/docs/lighthouse/performance/efficient-animated-content/)",
-      "score": 0,
+      "score": 0.5,
       "scoreDisplayMode": "metricSavings",
       "numericValue": 3450,
       "numericUnit": "millisecond",
       "displayValue": "Potential savings of 666 KiB",
       "metricSavings": {
         "FCP": 0,
-        "LCP": 3300
+        "LCP": 0
```

Comment:
This is a good change. Byte savings on the large gif are good, but it isn't the LCP resource so it's LCP impact is superficial.